.travis.yml

```diff
@@ -26,13 +30,19 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
+  - npm install -g bower phantomjs-prebuilt yarn
```

Comment:
I tried about 10 ways to run yarn and found some of the travis docs to not Just Work™ I did attempt two variations of an apt-get. Prolly can tune this up later. But the npm install does work. That was the easiest path to get started with Yarn, and Firefox and Chrome running the the build. I'd like to revisit this setup at some point.